pkg/operator/ceph/csi/template/rbd/csi-rbdplugin.yaml

```diff
@@ -68,6 +68,7 @@ spec:
             - "--metricspath=/metrics"
             - "--enablegrpcmetrics={{ .EnableCSIGRPCMetrics }}"
             - "--stagingpath={{ .KubeletDirPath }}/plugins/kubernetes.io/csi/pv/"
+            - "--csi-addons-address=$(CSI_ADDONS_ENDPOINT)"
```

Comment:
You have to create env var for this container too, env vars are not shared between containers. 
Same change is required for csi-rbdplugin-provisioner.yaml

Reply:
pass cmdline arg only if CSIADDONS is enabled.